packages/veritone-react-common/src/styles/modules/muiTypography.js

```diff
@@ -0,0 +1,79 @@
+// https://github.com/muicss/mui/blob/master/src/sass/mui/mixins/_typography.scss
+
+export const muiTextStyles = {
```

Comment:
Set this props to theme

Reply:
work like this?
[https://material-ui.com/customization/theming/#custom-variables](url)